src/core/LabeledTextarea/LabeledTextarea.tsx

```diff
@@ -98,7 +99,15 @@ export const LabeledTextarea = React.forwardRef<
       )}
       style={style}
     >
-      {label && <div className='iui-label'>{label}</div>}
+      {label && (
+        <div
+          className={cx('iui-label', {
+            'iui-required': required,
+          })}
+        >
+          {label}
+        </div>
+      )}
       <Textarea
         disabled={disabled}
         className={textareaClassName}
```

Comment:
Dont you need to add required on textarea???

Reply:
Done